docs/section21/Rule21-3.md

```diff
@@ -0,0 +1,53 @@
+
+# Boiler - Rule 21-3  
+
+**Rule ID:** 21-3  
+**Rule Description:** Heating hot water plant capacity shall be based on coincident loads.  
+**Rule Assertion:** B-RMR FluidLoop.heating_design_and_control: is_sized_using_coincident_load = True  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.2.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B-RMR  
+**Applicability Checks:**  
+
+1. Pass Rule 21-1.
+2. B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11, 12 or that is Type-1, 5, 7, 11, 12.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:**  
+
+1. check_purchased_chw_hhw()
+
+**Applicability Check:**
+
+1. Pass Rule 21-1: `if rule-21-1 == "PASS":`
+
+2. B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11, 12 or that is Type-1, 5, 7, 11, 12:
+
+  - Check if P-RMR is modeled with purchased cooling or purchased hot water/steam: `purchased_chw_hhw_status_dict = check_purchased_chw_hhw(P_RMR)`
+
```

Comment:
I think this function call should be moved to Line 25.

Reply:
Updated.

Reply:
I think Lines #33 - 43 can be replaces with 
 - Check if B-RMR is modeled with at least one air-side system that is Type-1a, 1, 5, 7, 11 or 12, continue to rule logic: `PLACEHOLDER`

    - Else, rule is not applicable: `else: rule_applicability_flag = FALSE`

Reply:
as discussed on the RDS call now we do not need to check if the P RMR is modeled with P cooling, we can simply reference the function and check if the baseline system type is 1a, 7a, etc. etc.

Reply:
good decision to add the CZ  check for System 11.